### PR TITLE
Changing Fx_version to more consistent RuntimeFrameworkVersion

### PR DESCRIPTION
**Pull request by ramarag**
Fix towards https://github.com/dotnet/sdk/issues/961

cc @eerhardt @nguerrera

### CODE REVIEW

File: src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets [hunk 1]
```diff
@@ -178,7 +178,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <!--
     ============================================================
                                         _GenerateCrossgenProj
-    Generates the project used to restore netcoreapp correspoding to FX_Version
+    Generates the project used to restore netcoreapp correspoding to RuntimeFrameworkVersion
```
Comment (nguerrera) [inline src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets]: typo: s/correspoding/corresponding/

File: src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets [hunk 2]
```diff
@@ -192,7 +192,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <PropertyGroup>
       <TargetFramework>$(_TFM)</TargetFramework>
       <RuntimeIdentifier>$(RuntimeIdentifier)</RuntimeIdentifier>
-      <RuntimeFrameworkVersion Condition="'$(FX_Version)' != ''">$(FX_Version)</RuntimeFrameworkVersion>
+      <RuntimeFrameworkVersion Condition="'$(RuntimeFrameworkVersion)' != ''">$(RuntimeFrameworkVersion)</RuntimeFrameworkVersion>
```
Comment (nguerrera) [inline src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets]: I don't see how this line does anything. It either does nothing when RuntimeFrameworkVErsion is empty or sets RuntimeFrameworkVersion to itself.

**Review by nguerrera — CHANGES_REQUESTED** [review]
See above

File: src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets (hunk 2 above)
Comment (nguerrera) [inline src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets]: Same for RuntimeIdentifier just above it.

Comment (nguerrera) [inline src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets]: Ah never mind. I see this is CDATA for generating a project. Funky and easy to miss in a diff with only so much context.

Comment (nguerrera) [inline src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets]: Wait, wouldn't the CDATA just write `$(RuntimeIdentifier)` expression and not its value and therefore generate a no-op after all. This is quite confusing.

Comment (ramarag) [inline src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets]: it is part of CDATA, used to generate a csproj

Comment (nguerrera) [inline src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets]: So the $(...) in CDATA is replaced with evaluated value? I see. The Condition is weird then. Can you remove it?

Comment (ramarag) [inline src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets]: done

**Review by nguerrera — APPROVED** [review]

**Review by eerhardt — APPROVED** [review]

**Comment by ramarag** [comment]
@dotnet-bot  test Ubuntu16.04

**Comment by ramarag** [comment]
@dotnet-bot  test Ubuntu16.04 Debug

**Comment by ramarag** [comment]
@dotnet-bot test Ubuntu16.04 Release

**Pull request merged**